client/src/component/MilestoneTable.js

```diff
@@ -0,0 +1,10 @@
+export function MilestoneTable() {
+
+	return (
+			<div>
+				<h1>hello world</h1>
+			</div>
+	);
+}
```

Comment:
Keep an eye on your formating. We have [pretier](https://prettier.io/) set up in the repo to get it to format a file press "shift, option, f" and have a look at the changes it makes

Reply:
thank you I will try this on next pull request